vllm/entrypoints/openai/api_server.py

```diff
@@ -127,11 +126,10 @@ async def build_backend(args) -> AsyncIterator[VLLMBackend]:
             rpc_server_process.terminate()
 
             # Close all open connections to the backend
-            logger.info("Cleaning up ZMQ client context")
             backend.close()
 
-            # Wait for server process to join
-            rpc_server_process.join()
+        # Wait for server process to join
+        rpc_server_process.join()
```

Comment:
I actually meant this to be in the `finally` statement so that it will run on exit when there's an unhandled exception, which can happen if there's an exception before we get to the guarded ```
tyr:
  await server_task
```

Reply:
this was a typo, will fix